network_insight_sdk_generic_datasources/common/physical_device.py

```diff
@@ -91,7 +95,14 @@ def execute_commands(self):
                     py_logger.info('Command %s Result %s' % (workload[REUSE_COMMAND_KEY], command_result))
                     table = self.parse_command_output(workload, command_result)
                 else:
-                    command_result = ssh_connect_handler.execute_command(workload[COMMAND_KEY])
+                    if self.file_input:
+                        # In case of file input, read from the file named as <table_id of command in yml file>.txt
+                        # File is read from the file input directory specified in the yml file
+                        file_path = self.file_input_directory + '/' + command_id + '.txt'
+                        f = open(file_path, 'r')
+                        command_result = f.read() if f.mode == 'r' else ''
+                    else:
+                        command_result = ssh_connect_handler.execute_command(workload[COMMAND_KEY])
```

Comment:
Can we refactor lines 79-105 ? Idea is to make it more readable. 
execute_commands() method should be called for ssh command executions only. For file processing, we need to have separate methods.

Reply:
Addressed review comment in https://github.com/vmware/network-insight-sdk-generic-datasources/pull/49/commits/6a3b645b5553fe3f5fb23513962680c5b5829d94